### PR TITLE
Update bluetooth article with latest news from M56

### PR DESCRIPTION
**Pull request by beaufortfrancois**
What's new?

- `new Uint8Array([...])` -> `Uint8Array.of(...)`
- `acceptAllDevices` key
- shim in noble

R: @g-ortuno @scheib

### CODE REVIEW

File: src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md [hunk 1]
```diff
@@ -372,6 +390,7 @@ Check out our [curated Web Bluetooth Demos](https://github.com/WebBluetoothCG/de
 ## Libraries
 
 - [web-bluetooth-utils](https://www.npmjs.com/package/web-bluetooth-utils) is a npm module that adds some convenience functions to the API.
+- A Web Bluetooth API shim is available in [noble](https://github.com/sandeepmistry/noble), the most popular Node.js BLE central module. This allows you to webpack/browserify noble without the need for a WebSocket server or other plugins.
```
Comment (scheib) [inline src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md]: I don't understand what "allows you to webpack/browserify noble without ..." means. What's fixing what? I would have guessed this means that devs can use web bluetooth style code in non-browsers. Basically, what I can do with this that I couldn't before?

Comment (g-ortuno) [inline src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md]: I think it is the other way around. This basically allows you to run your noble code on the web.

Comment (beaufortfrancois) [inline src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md]: With this new Web Bluetooth API shim, developers can now webpack/browserify noble - like they used to do to run it on the web - but this time without WebSocket bindings. See https://github.com/sandeepmistry/noble/pull/238/files

@monteslu will confirm for sure.

Comment (monteslu) [inline src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md]: Correct, people can now use the noble API (BLE for node.js) in a browser.

Comment (TheLarkInn) [inline src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md]: So this is a webpack'able library is what I'm reading. Correct me if I'm wrong (sorry a bit late to this party [but part of the webpack team and wanted to make sure we catch any related questions in our github if they come up])

Comment (beaufortfrancois) [inline src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md]: ping @monteslu

**Pull request merged**